WcaOnRails/spec/requests/api_persons_spec.rb

```diff
@@ -53,4 +53,14 @@
       expect(json["personal_records"]["333"]["average"]["best"]).to eq 590
     end
   end
+
+  describe "GET #results" do
+    it "renders properly" do
+      get api_v0_person_results_path(person.wca_id)
+      expect(response).to be_successful
+      json = JSON.parse(response.body)
+      expect(json[0]["personId"]).to eq person.wca_id
+      expect(json[0]["personName"]).to eq person.name
```

Comment:
Ah this reminded me that results have camelCased attributes (that's because it's an old database table and some PHP code still relies on those attributes), whereas we use camel_case for other endpoints. We can fix that by defining `serializable_hash` in `Result` (`models/result.rb`) as well. I'm thinking something like that:
```rb
  def serializable_hash(options = nil)
    {
      id: id,
      pos: pos,
      event_id: eventId,
      round_type_id: roundTypeId,
      format_id: formatId,
      attempts: [value1, value2, value3, value4, value5],
      best: best,
      average: average,
      regional_single_record: regionalSingleRecord,
      regional_average_record: regionalAverageRecord
    }
  end
```
Additionally I omitted person related data, as it's wasteful to repeat that in every object in the resulting array, and you can get that (and more) data from the person endpoint.

Reply:
This is _so_ pedantic, but I figured I'd point it out: `camelCase` vs `snake_case` :laughing:

Reply:
Personally I'd be up for `camelCase`, but until we have some discussion on that we should be consistent with all the existing code ^^